sqlglot/generator.py

```diff
@@ -209,6 +209,16 @@ def table_sql(self, expression):
     def tuple_sql(self, expression):
         return f"({self.expressions(expression, flat=True)})"
 
+    def update_sql(self, expression):
+        this = self.sql(expression, 'this')
+        set_exp = expression.args.get('set')
+        set_sql = self.sql(set_exp)
```

Comment:
probably cleaned up a bit

set_sql = self.expressions(expression)

Reply:
Sure, I've cleaned it up.